charts/elasti/templates/_helpers.tpl

```diff
@@ -57,6 +132,340 @@ Image pull secrets
 {{- toYaml .Values.global.imagePullSecrets }}
 {{- end }}
 
+{{/*
+===========================================
+Elasti Operator Helpers
+===========================================
+*/}}
+
+{{/*
+Expand the name of the elasti-operator component
+*/}}
+{{- define "elasti-operator.name" -}}
+{{- default "elasti-operator" .Values.elastiController.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name for elasti-operator
+*/}}
+{{- define "elasti-operator.fullname" -}}
+{{- if .Values.elastiController.fullnameOverride }}
+{{- .Values.elastiController.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default "elasti-operator" .Values.elastiController.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Common labels for elasti-operator - uses global elasti.labels function
+*/}}
+{{- define "elasti-operator.labels" -}}
+{{- include "elasti.labels" (dict "context" . "name" "elasti-operator") }}
+{{- end }}
+
+{{/*
+Common labels - merges base labels, global labels, and component-specific labels
+Priority (highest to lowest): elastiController.commonLabels > global.labels > base labels
+*/}}
+{{- define "elasti-operator.commonLabels" -}}
+{{- $baseLabels := include "elasti-operator.labels" . | fromYaml }}
+{{- $mergedLabels := mergeOverwrite $baseLabels (deepCopy .Values.global.labels) .Values.elastiController.commonLabels }}
```

Comment:
Why are we including `.Values.global.labels` if we are already having 
```
{{- with .context.Values.global.labels }}
{{ toYaml . }}
```
in the `elasti.labels` function